docs/getting-started/croeseid-testnet.md

```diff
@@ -100,8 +106,7 @@ You should obtain an address with `tcro` prefix, e.g. `tcro1quw5r22pxy8znjtdkgqc
 
 ### Step 3-2. Obtain test token
 
-Unless you have obtained the CRO testnet token before, simply send a message on [Gitter](https://gitter.im/crypto-com/community),
-stating who you are and your `tcro.....` address (@devashishdxt or @lezzokafka would typically reply within a day).
+Unless you have obtained the CRO testnet token before, use the [CRO faucet](https://chain.crypto.com/faucet) to obtain test CRO tokens.
```

Comment:
Would have to keep this as the faucet would only give them a small amount per day.

Reply:
Yeah, sure, I'm putting it back, I will mention that it will be needed in cases the receiver has exceeded the daily faucet airdrop limit.